src/api/window/window.ts

```diff
@@ -903,14 +917,17 @@ export class _Window extends EmitterBase<WindowEvents> {
      * @param { AnchorType } anchor Specifies a corner to remain fixed during the resize.
      * Can take the values: "top-left", "top-right", "bottom-left", or "bottom-right".
      * If undefined, the default is "top-left"
+     * @param { WindowMovementOptions } options Optional parameters to modify window movement
      * @return {Promise.<void>}
      * @tutorial Window.resizeBy
      */
-    public resizeBy(deltaWidth: number, deltaHeight: number, anchor: AnchorType): Promise<void> {
+    public resizeBy(deltaWidth: number, deltaHeight: number, anchor: AnchorType,
+        options: WindowMovementOptions = { moveIndependently: true }): Promise<void> {
```

Comment:
why default true here and false elsewhere?

Reply:
Super weird, totally missed this. Fixed, thanks!